darshan_logs/hdf5_diagonal_write_only/README.md

```diff
@@ -0,0 +1,22 @@
+Set of example darshan logs that contain `H5D` and/or `H5F` module data
+to allow for "ground truth" verification for various data aggregation
+functions in the new Python summary report.
+
+The example application code used to generate these logs along with
+some of the execution details are available here:
+https://github.com/nawtrey/darshan_hdf5_testing
+
+A brief summary of each log's contents:
+
+| Log                                   | Modules                            | Description                                      |
+|---------------------------------------|------------------------------------|--------------------------------------------------|
+| `hdf5_file_opens_only`                | `POSIX`, `H5F`                     | 3 ranks open files                               |
+| `hdf5_diagonal_write_1_byte_dxt`      | `POSIX`, `DXT_POSIX`, `H5F`, `H5D` | 10 ranks open files, write 1 byte each           |
+| `hdf5_diagonal_write_bytes_range_dxt` | `POSIX`, `DXT_POSIX`, `H5F`, `H5D` | 10 ranks open files, write 10-100 bytes each     |
+| `hdf5_diagonal_write_half_flush_dxt`  | `POSIX`, `DXT_POSIX`, `H5F`, `H5D` | 10 ranks open files, write 1 byte, 5 ranks flush |
+| `hdf5_diagonal_write_half_ranks_dxt`  | `POSIX`, `DXT_POSIX`, `H5F`, `H5D` | 10 ranks open files, 5 ranks write 1 byte        |
+
+
+These were created on a single node of the LANL snow supercomputer. The
+commit hash of the darshan main branch used to build `darshan-runtime`
+is `c805d948`, and it was compiled with the `HEATMAP` module disabled.
```

Comment:
Is there a particular reason the heatmap was disabled?  Does it's presence cause a problem for the pydarshan stuff at the moment?

If not it's fine, and we can just close this review comment.  I just wanted to make sure there wasn't a particular rationale that we should document or if that's just the way it happened to be built.

Reply:
I turned it off because I didn't think it was necessary to keep the `HEATMAP` module data for this use case. It was also a little difficult to test the logs on my local HDF5 feature branches without rebasing them on Jakob's branch with the bindings (at the time it wasn't merged), but that's not a particularly strong reason for leaving it out. It was mainly because I thought it was unnecessary, I wanted the logs to be as lightweight as possible so they can be loaded and tested quickly. If we want the `HEATMAP` data there I can certainly regenerate them though.

Reply:
Ok, no problem as is, I just wanted to make sure there wasn't a specific issue to document.  Thanks!